main.go

```diff
@@ -23,38 +27,46 @@ var postgresFunctions = []runtime.Transform{
 	},
 }
 
-func main() {
-	app := newApp()
-	err := app.Run(os.Args)
-	// If required flags aren't set, it will return with error before we could set up logging
-	if err != nil {
-		_, _ = fmt.Fprintf(os.Stderr, "%v\n", err)
-		os.Exit(1)
-	}
+var (
+	Network = "unix"
+	Address = "@crossplane/fn/default.sock"
+	// for testing purposes, especially on MacOS it's much easier to create local socket than whole directory structure and permissions
+	//Address = "default.sock"
+)
```

Comment:
Please make this configurable through flags. That way you can also remove the commented code